prody2/protocols/protocol_modes_base.py

```diff
@@ -135,6 +135,8 @@ def _insertAllSteps(self, n):
         # Insert processing steps
         #n = self.numberOfModes.get()
 
+        self.gnm = False
+
         self._insertFunctionStep('computeModesStep')
```

Comment:
FYI: steps can be passed as methods as well as strings.

`self._insertFunctionStep(self.computeModesStep)`

Reply:
ok. Good to know. Thanks